scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/JobEmailNotification.java

```diff
@@ -44,8 +44,12 @@ public JobEmailNotification(JobState js, NotificationData<JobInfo> notification)
 
     public boolean doCheckAndSend() throws JobEmailNotificationException {
         switch (eventType) {
+            case JOB_PAUSED:
+            case JOB_RESUMED:
+            case JOB_IN_ERROR:
```

Comment:
Personally, I would add as well JOB_SUBMITTED, JOB_PENDING_TO_RUNNING, JOB_CHANGE_PRIORITY. 

When I worked with CCRT SLURM scheduler, I was notified by email when a job was submitted and when it moved from the pending queue to the running queue. I suppose as well the user wants to be notified if an admin has changed the priority of his job. I don't know about the event JOB_UPDATED, I can't figure out what it's doing.